internship2018-1/src/arrayFilters.js

```diff
@@ -0,0 +1,34 @@
+export function filterArray(numbers, min, max){
+    var counter = 0;
+    var result = [];    
+    for(var i = 0; i < numbers.length; i++){
+        if(numbers[i] > min && numbers[i] < max){            
+            result[counter] = numbers[i];
+            counter++;
+        }
+    }
+    return result;
+}
+
+export function isEmpty(object){
+    if (Object.keys(object).length > 0){
+        return false;
+    }
+    else {
```

Comment:
The else statement should be placed on the same line as the closing curly brace:

if (expression) {
...
} else {
...
}